examples/p4est_2d_dgsem/elixir_advection_diffusion_periodic_curved.jl

```diff
@@ -0,0 +1,88 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linear advection-diffusion equation
+
+diffusivity() = 5.0e-2
+advection_velocity = (1.0, 0.0)
+equations = LinearScalarAdvectionEquation2D(advection_velocity)
+equations_parabolic = LaplaceDiffusion2D(diffusivity(), equations)
+
+function x_trans_periodic(x, domain_length = SVector(2 * pi), center = SVector(0.0))
```

Comment:
```suggestion
function x_trans_periodic(x, domain_length=SVector(2 * pi), center=SVector(0.0))
```
Same as before.

Reply:
See other comment

Reply:
This is exactly what you changed in the other example file.

Reply:
Oops! I misread it. Thanks